src/component/organism/form/PhoneNumVerification.js

```diff
@@ -10,8 +10,7 @@ import Input30 from 'Molecules/input/Input30';
 import InputTimeLimit from 'Molecules/input/InputTimeLimit';
 import InputWithSelect from 'Molecules/input/InputWithSelect';
 import {btn_style, temp_style} from 'Templete/style_templete';
-import {phoneNumVerification} from 'Organism/style_organism';
-
+import {phoneNumVerification} from '../style_organism copy';
```

Comment:
import 시에 상대참조보다 절대참조로 가도록 변경 'Organism/style_organism copy'로 변경해야함
```suggestion
import {phoneNumVerification} from 'Organism/style_organism copy';
```